### PR TITLE
MIgrate to Swift 4

### PR DESCRIPTION
**Pull request by danielpassos**

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -5,5 +5,5 @@ platform :ios, '9.0'
 use_frameworks!
 
 target 'blank-ios-app' do
-	pod 'FeedHenry', '5.0.5'
+	pod 'FeedHenry', '6.0.0'
```
Comment (jcesarmobile) [inline Podfile]: On the Objective-C ones we have `~>`, so I'm using it on the Swift ones too

**Comment by jcesarmobile** [comment]
I've added a comment. 
Also, can you update the README.md with the new SDK version, Xcode version and some other changes like the title (it says `blank-ios-app` but this is the swift one), the feedhenry link to the config no longer works, the new one is https://access.redhat.com/documentation/en-us/red_hat_mobile_application_platform_hosted/3/html/client_sdk/native-ios-swift, and some other things you might see.

**Comment by danielpassos** [comment]
@jcesarmobile Done, can you double check?!

File: README.md [hunk 2]
```diff
@@ -1,24 +1,23 @@
 # blank-ios-app
```
Comment (jcesarmobile) [inline README.md]: Title should be blank-ios-swift

File: README.md [hunk 3]
```diff
@@ -1,24 +1,23 @@
...
 Level: Intermediate   
-Technologies: Swift 3, iOS, RHMAP, CocoaPods.   
+Technologies: Swift 4, iOS, RHMAP, CocoaPods.   
 Summary: A demonstration of how to get started with RHMAP.   
 Community Project : [Feed Henry](http://feedhenry.org)   
 Target Product: RHMAP   
 Product Versions: RHMAP 3.7.0+   
 Source: https://github.com/feedhenry-templates/blank-ios-swift   
-Prerequisites: fh-ios-swift-sdk: 5+, Xcode: 8+, iOS SDK: iOS 9+, CocoaPods: 1.3.0+
+Prerequisites: fh-ios-swift-sdk: 5+, Xcode: 9+, iOS SDK: iOS 9+, CocoaPods: 1.3.0+
```
Comment (jcesarmobile) [inline README.md]: fh-ios-swift-sdk should be 6+

**Review by jcesarmobile — CHANGES_REQUESTED** [review]
A few more changes

**Review by jcesarmobile — CHANGES_REQUESTED** [review]
Added a few comments

File: README.md (hunk 3 above)
Comment (danielpassos) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (danielpassos) [reply README.md]: Done!

**Review by jcesarmobile — APPROVED** [review]
LGTM

**Pull request merged**